deploy/terraform-datacommons-website/test/go.mod

```diff
@@ -0,0 +1,89 @@
+module github.com/datacommonsorg/website/test
+
+go 1.19
+
+require (
```

Comment:
can you try `go mod tidy` to clean the modules here

Reply:
This is the result from go mod tidy. Do you mean that you think there are more indirect dependencies then there should be?

Reply:
Yea, but maybe terraform does use a lot of libraries.